deploy/install.sh

```diff
@@ -34,10 +34,10 @@ fi
 
 echo "MOSIP_INJIWEB_HOST is not present in configmap/global of configserver"
     # Add injiweb host to global
-    kubectl patch configmap global -n config-server --type merge -p "{\"data\": {\"mosip-injiweb-host\": \"$MOSIP_INJIWEBB_HOST\"}}"
-    kubectl patch configmap global -n default --type merge -p "{\"data\": {\"mosip-injiweb-host\": \"$MOSIP_INJIWEBB_HOST\"}}"
+    kubectl patch configmap global -n config-server --type merge -p "{\"data\": {\"mosip-injiweb-host\": \"$MOSIP_INJIWEB_HOST\"}}"
```

Comment:
seems the configmap name was changed in mimoto. Hence same should be rflecting here.